### ISSUE TITLE
JsonException when converting value to Camille.Enums.Division

### ISSUE THREAD

**Issue by aPinat** [MingweiSamuel/Camille#46]
Happens in latest 3.0.0-nightly-2020-09-02-2ad01835f7 during `LeagueV4().GetLeagueEntriesForSummonerAsync()` not sure about other functions...
Latest working version seams to be 3.0.0-nightly-2020-08-27-1543d8fc57.

```
System.Text.Json.JsonException: The JSON value could not be converted to Camille.Enums.Division. Path: $[0].rank | LineNumber: 0 | BytePositionInLine: 108.
   at System.Text.Json.ThrowHelper.ThrowJsonException(String message)
   at System.Text.Json.Serialization.Converters.EnumConverter`1.Read(Utf8JsonReader& reader, Type typeToConvert, JsonSerializerOptions options)
   at System.Text.Json.JsonPropertyInfo`1.ReadJsonAndSetMember(Object obj, ReadStack& state, Utf8JsonReader& reader)
   at System.Text.Json.Serialization.Converters.ObjectDefaultConverter`1.OnTryRead(Utf8JsonReader& reader, Type typeToConvert, JsonSerializerOptions options, ReadStack& state, T& value)
   at System.Text.Json.Serialization.JsonConverter`1.TryRead(Utf8JsonReader& reader, Type typeToConvert, JsonSerializerOptions options, ReadStack& state, T& value)
   at System.Text.Json.Serialization.Converters.IEnumerableDefaultConverter`2.OnTryRead(Utf8JsonReader& reader, Type typeToConvert, JsonSerializerOptions options, ReadStack& state, TCollection& value)
   at System.Text.Json.Serialization.JsonConverter`1.TryRead(Utf8JsonReader& reader, Type typeToConvert, JsonSerializerOptions options, ReadStack& state, T& value)
   at System.Text.Json.Serialization.JsonConverter`1.ReadCore(Utf8JsonReader& reader, JsonSerializerOptions options, ReadStack& state)
   at System.Text.Json.JsonSerializer.ReadCore[TValue](Utf8JsonReader& reader, Type returnType, JsonSerializerOptions options)
   at System.Text.Json.JsonSerializer.Deserialize[TValue](String json, Type returnType, JsonSerializerOptions options)
   at Camille.RiotGames.RiotGamesApi.Send[T](String route, String methodId, HttpRequestMessage request, Nullable`1 token, Boolean ignoreAppRateLimits)
```

**Comment by aPinat** [comment]
Referencing any of the build that uses `Newtonsoft.Json` instead of `System.Text.Json` still works fine.
`netcoreapp3.1` is being defaulted to for me and is not using `Newtonsoft.Json`.

**Comment by MingweiSamuel** [comment]
Ah sorry looks like this one slipped through my notifications. Will take a look this afternoon

**Comment by MingweiSamuel** [comment]
Broke it in 3ba2a6d0186415acbb43f2d413ec5862c28314ff
Fixed in  7045b7a

Released https://www.nuget.org/packages/Camille.RiotGames/3.0.0-nightly-2020-11-01-7045b7aeda w/ Seraphine & this bugfix
